index.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+    />
+    <link rel="stylesheet" href="css/styles.css" />
+
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <script src="https://cdn.tailwindcss.com"></script>
+  </head>
+
+  <body>
+    <div id="container">
+      <header
+        id="header"
+        class="fixed p-8 px-12 bg-gray-800 text-white flex items-center justify-between w-screen"
+      >
+        <i class="fas fa-gamepad"> Joysticks </i>
+        <nav id="nav-bar">
+          <ul class="flex items-center gap-12">
+            <li><a class="nav-link" href="#about">About</a></li>
+            <li><a class="nav-link" href="#types">Types</a></li>
+            <li><a class="nav-link" href="#pricing">Pricing</a></li>
+            <li><a class="nav-link" href="#form">Form</a></li>
+          </ul>
+        </nav>
+      </header>
+
+      <div class="h-28"></div>
+
+      <div class="mb-20 heading w-full flex items-center justify-center">
+        <h1 class="text-3xl font-bold">First Choice Joysticks</h1>
+      </div>
+
+      <div id="about" class="px-24">
+        <div class="flex flex-col gap-4">
+          <h2 class="text-2xl font-bold">What We offer</h2>
+          <div class="text-gray-600">
+            <p>
+              We have a range of joysticks for all available consoles for your
+              enjoyment.
+            </p>
+            <p>
+              We only have top quality products that are guaranteed to provide
+              you with long lasting fun with all your games
+            </p>
+          </div>
+        </div>
+      </div>
+
```

Comment:
And also here. I would recommend you use `<section><section/>` tags for different sections on your website for every different sections.